package.json

```
@@ -1,53 +1,60 @@
 {
-    "name": "interactive-examples",
-    "version": "2.0.0",
-    "description": "Interactive code examples embedded on MDN",
-    "author": "Mozilla",
-    "license": "MPL-2.0",
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/mdn/interactive-examples.git"
-    },
-    "bugs": {
-        "url": "https://github.com/mdn/interactive-examples/issues"
-    },
-    "homepage": "https://github.com/mdn/interactive-examples#readme",
-    "main": "index.js",
-    "scripts": {
-        "build": "mdn-bob",
-        "build:pages": "mdn-bob --skip-webpack",
-        "fix:css": "npm run lint:css -- --fix",
-        "fix:js": "eslint --fix ./live-examples/js-examples",
-        "lint:css": "stylelint \"**/*.css\"",
-        "lint:js": "eslint ./live-examples/js-examples",
-        "postinstall": "npm run build",
-        "start-server": "http-server -p 9090 ./docs",
-        "start-watch": "chokidar \"./live-examples/**\" -c \"npm run build:pages\" --initial --silent",
-        "start": "npm-run-all --parallel start-watch start-server",
-        "test": "npm run lint:js"
-    },
-    "keywords": [
-        "javascript",
-        "css",
-        "nodejs",
-        "page-generator",
-        "mdn",
-        "mozilla"
-    ],
-    "engines": {
-        "node": ">=18.0.0"
-    },
-    "devDependencies": {
-        "@babel/core": "^7.22.9",
-        "@babel/eslint-parser": "^7.22.9",
-        "@mdn/bob": "^3.3.0",
-        "chokidar-cli": "^3.0.0",
-        "eslint": "^8.45.0",
-        "http-server": "^14.1.1",
-        "npm-run-all": "^4.1.5",
-        "prettier": "^2.8.8",
-        "prettier-eslint": "^15.0.1",
-        "stylelint": "^15.10.2",
-        "stylelint-prettier": "^3.0.0"
-    }
+  "name": "interactive-examples",
+  "version": "2.0.0",
+  "description": "Interactive code examples embedded on MDN",
+  "author": "Mozilla",
+  "license": "MPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mdn/interactive-examples.git"
+  },
+  "bugs": {
+    "url": "https://github.com/mdn/interactive-examples/issues"
+  },
+  "homepage": "https://github.com/mdn/interactive-examples#readme",
+  "main": "index.js",
+  "scripts": {
+    "build": "mdn-bob",
+    "build:pages": "mdn-bob --skip-webpack",
+    "fix": "npm run fix:css && npm run fix:js && npm run fix:format",
+    "fix:css": "npm run lint:css -- --fix",
+    "fix:js": "eslint --fix ./live-examples/js-examples",
```

```suggestion
    "fix:js": "eslint --fix .",
```
Then add `docs/*` to `.eslintignore` file.

At the moment this is linting only `js-examples`. [[ref](https://github.com/mdn/interactive-examples/pull/2561#issuecomment-1652848342)]